### ISSUE TITLE
Refer to the installation tutorial, use the main branch,  model training is slow

### ISSUE THREAD

**Issue by zhangzscn** [open-mmlab/mmdetection#10329]
Using the COCO dataset, eight sheets of a40 to train the rtmdet-ins_tiny model takes 15 days。

2023/05/15 13:38:13 - mmengine - INFO - 
------------------------------------------------------------
System environment:
    sys.platform: linux
    Python: 3.8.16 (default, Mar  2 2023, 03:21:46) [GCC 11.2.0]
    CUDA available: True
    numpy_random_seed: 1038074250
    GPU 0,1,2,3,4,5,6,7: NVIDIA A40
    CUDA_HOME: /usr/local/cuda
    NVCC: Cuda compilation tools, release 11.3, V11.3.109
    GCC: gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
    PyTorch: 2.0.1
    PyTorch compiling details: PyTorch built with:
  - GCC 9.3
  - C++ Version: 201703
  - Intel(R) oneAPI Math Kernel Library Version 2023.1-Product Build 20230303 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.7.3 (Git Hash 6dbeffbae1f23cbbeae17adb7b5b13f1f37c080e)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.8
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_90,code=sm_90;-gencode;arch=compute_37,code=compute_37
  - CuDNN 8.7
  - Magma 2.6.1
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.8, CUDNN_VERSION=8.7.0, CXX_COMPILER=/opt/rh/devtoolset-9/root/usr/bin/c++, CXX_FLAGS= -D_GLIBCXX_USE_CXX11_ABI=0 -fabi-version=11 -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -DNDEBUG -DUSE_KINETO -DLIBKINETO_NOROCTRACER -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -O2 -fPIC -Wall -Wextra -Werror=return-type -Werror=non-virtual-dtor -Werror=bool-operation -Wnarrowing -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wunused-local-typedefs -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Werror=cast-function-type -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_DISABLE_GPU_ASSERTS=ON, TORCH_VERSION=2.0.1, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF, 

    TorchVision: 0.15.2
    OpenCV: 4.7.0
    MMEngine: 0.7.3

Runtime environment:
    cudnn_benchmark: False
    mp_cfg: {'mp_start_method': 'fork', 'opencv_num_threads': 0}
    dist_cfg: {'backend': 'nccl'}
    seed: None
    Distributed launcher: pytorch
    Distributed training: True
    GPU number: 8
------------------------------------------------------------

2023/05/15 13:47:26 - mmengine - INFO - Epoch(train)   [1][ 50/459]  lr: 1.9623e-04  eta: 15 days, 21:43:04  time: 9.9832  data_time: 7.0772  memory: 15938  loss: 3.7053  loss_cls: 0.8942  loss_bbox: 1.1795  loss_mask: 1.6316
2023/05/15 13:55:14 - mmengine - INFO - Epoch(train)   [1][100/459]  lr: 3.9643e-04  eta: 15 days, 9:28:09  time: 9.3495  data_time: 2.7249  memory: 17710  loss: 2.5097  loss_cls: 0.8399  loss_bbox: 0.8998  loss_mask: 0.7701
2023/05/15 14:02:54 - mmengine - INFO - Epoch(train)   [1][150/459]  lr: 5.9663e-04  eta: 15 days, 3:24:42  time: 9.2013  data_time: 1.4395  memory: 18459  loss: 2.4877  loss_cls: 0.9482  loss_bbox: 0.8592  loss_mask: 0.6803
2023/05/15 14:10:47 - mmengine - INFO - Epoch(train)   [1][200/459]  lr: 7.9683e-04  eta: 15 days, 2:54:36  time: 9.4726  data_time: 0.2362  memory: 18301  loss: 2.1678  loss_cls: 0.8899  loss_bbox: 0.7216  loss_mask: 0.5563
2023/05/15 14:18:00 - mmengine - INFO - Epoch(train)   [1][250/459]  lr: 9.9703e-04  eta: 14 days, 20:14:14  time: 8.6451  data_time: 0.0037  memory: 18655  loss: 2.0288  loss_cls: 0.8471  loss_bbox: 0.6662  loss_mask: 0.5154
2023/05/15 14:25:35 - mmengine - INFO - Epoch(train)   [1][300/459]  lr: 1.1972e-03  eta: 14 days, 18:43:49  time: 9.1138  data_time: 0.0034  memory: 18493  loss: 2.0216  loss_cls: 0.8476  loss_bbox: 0.6588  loss_mask: 0.5152
2023/05/15 14:32:56 - mmengine - INFO - Epoch(train)   [1][350/459]  lr: 1.3974e-03  eta: 14 days, 15:57:32  time: 8.8094  data_time: 0.0036  memory: 18736  loss: 1.9782  loss_cls: 0.8297  loss_bbox: 0.6428  loss_mask: 0.5057
